src/models/Transaction.js

```diff
@@ -24,7 +24,11 @@ export type TxSettlementItem = {
   hash: string,
 };
 
-export type TransactionExtra = TxSettlementItem[];
+export type TxWithdrawalExtra = {
```

Comment:
I wouldn't export symbols by default, since linters can't help detect unused stuff for exported names. This one looks like is not being used outside this file.